data/schedule.json

```diff
@@ -10,28 +10,44 @@
                 {
                     "Time": "10:00 AM - 01:00 PM",
                     "Tracks": [{
+                            "sno":"1",
```

Comment:
@akshat0047 What is the need of `sno` here ?

Reply:
sno is used to give id's to popup so that on the basis of it their respective block can be popped.

Reply:
I prefer not to put css ids in the json data. Can you do this using a counter in js code ?